packages/plugin-xml/src/Attributes.ts

```diff
@@ -1,17 +1,34 @@
 import { Modifier } from '../../core/src/Modifier';
+import { CssStyle } from './CssStyle';
+import { ClassList } from './ClassList';
 
 export class Attributes extends Modifier {
     private _record: Record<string, string> = {};
-    constructor(attributes?: Record<string, string> | Attributes) {
+    style = new CssStyle();
+    classList = new ClassList();
```

Comment:
I think I prefer `classList` because (1) it's more coherent with the DOM and (2) with `class` I think I'd expect it to be a string, not a set of strings.